packages/client/ui/react-ui/src/index.ts

```diff
@@ -3,4 +3,5 @@ export * from "./hooks";
 export * from "./providers";
 
 export { CrossmintEvents, useCrossmintEvents } from "@crossmint/client-sdk-base";
+export type { EVMSmartWallet, Chain } from "@crossmint/client-sdk-smart-wallet";
```

Comment:
Removing `@crossmint/client-sdk-smart-wallet` as a dependency from the demo package revealed we need to export these.